packages/react-jss/package.json

```diff
@@ -42,11 +42,13 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.0.0",
-    "@types/react": "^16.4.18",
     "hoist-non-react-statics": "^3.2.0",
     "jss": "^10.0.0-alpha.8",
     "jss-preset-default": "^10.0.0-alpha.8",
     "prop-types": "^15.6.0",
     "theming": "^3.0.2"
+  },
+  "devDependencies": {
```

Comment:
what about additionally having them in peerDependencies?

Reply:
This would require everyone to install them 🤔 
Or if they don't decide on installing them, they will constantly have a warning for a missing peer dependency which isn't nice either

Reply:
yeah, how do we ensure that userr gets the right version of typings with a version of JSS?

Reply:
I don't think we can.

Reply:
The things which we are using from `react` are also not to special/new so I don't think we will have a problem with that.

Reply:
The definetly typed version also has no dependency on react types package